### PR TITLE
QML learning path

### PR DESCRIPTION
**Pull request by frankharkins**
[Link to preview](https://platypus-pr-173.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/course/machine-learning/introduction)

> ~~**NOTE**: do not merge yet~~

---

Fix https://github.com/Qiskit/qiskit.org/issues/2228

### CODE REVIEW

**Comment by vabarbosa** [comment]
converting to to active PR so we can get updated previews. do not merge until ready

**Comment by vabarbosa** [comment]
a couple comments:

- should the **_IBM Quantum Lab_** text be a link to https://quantum-computing.ibm.com/lab?

    image

- the following links did not work

    image

    image

    image

    image

**Comment by frankharkins** [comment]
@vabarbosa thank you! Should be all fixed in d37b093 hopefully.

File: notebooks/toc.yaml
Comment (vabarbosa) [inline notebooks/toc.yaml]: just wanted to call this out. please be sure to properly restore the `toc.yaml` before merging this PR.

**Review by vabarbosa — CHANGES_REQUESTED** [review]
FYI, the tests are failing because most of the pages are missing from the build. this is due to the `toc.yaml` which is currently stripped of most content. be sure to properly restore the `toc.yaml` before merging the changes

File: notebooks/toc.yaml
Comment (frankharkins) [reply notebooks/toc.yaml]: Whoops sorry must have accidentally committed that

**Comment by vabarbosa** [comment]
@frankharkins @JRussellHuffman i believe this is good to go. should the **[WIP]** be removed and we proceed with merging this afternoon?

**Review by vabarbosa — APPROVED** [review]
thanks

**Pull request merged**
